_includes/code_snippet.md

```diff
@@ -5,19 +5,39 @@
 {{ code }}
 ```
 {% assign nanosecond = "now" | date: "%N" %}
-<textarea id="code{{ nanosecond }}" style="display:none;">{{ code | xml_escape }}</textarea>
-<button id="copybutton{{ nanosecond }}" data-clipboard-target="#code{{ nanosecond }}">
+<button id="copybutton{{ nanosecond }}" data-clipboard-text="{{ code | xml_escape }}" aria-label="Copied!">
   Copy to clipboard
 </button>
 
 <script>
-var copybutton = document.getElementById('copybutton{{ nanosecond }}');
-var clipboard{{ nanosecond }} = new Clipboard(copybutton);
+var clipboard{{ nanosecond }} = new ClipboardJS('#copybutton{{ nanosecond }}');
 
 clipboard{{ nanosecond }}.on('success', function(e) {
+
+    // Set Button HTML to 'Copied!'
+    document.querySelector('#copybutton{{ nanosecond }}').innerHTML = 'Copied!';
+
+    // After 2 seconds, return back to original text.
+    setTimeout(() => {
+     document.querySelector('#copybutton{{ nanosecond }}').innerHTML = 'Copy to clipboard';
+    }, 2000);
+
+    /* Testing custom logic through console
+    console.info('Action:', e.action);
```

Comment:
I understand needing the console.info lines for testing, but are they needed in production?